043-SQLModernization/Coach/README.md

```diff
@@ -0,0 +1,21 @@
+# What The Hack - SQL Modernization and Migration
+
+## Introduction
+Welcome to the coach's guide for the SQL Modernization and Migration What The Hack. Here you will find links to specific coaches' guidance for each of the challenges.
+
+The goal of each challenge is to be as loosely coupled as possible to support a modular approach. It may be possible to skip challenges, rearrange them, etc., with any dependencies spelled out in each challenge and coach guide. If you run into any problems, please open an issue so it can be looked at.
+
+## Presentations
+
+* [Migrating SQL Server to Azure](./Coach/SQLMigration.pptx)
```

Comment:
This link is broken.  You've got an extra "Coach" in the path.  Also, use a raw link here.

(./SQLMigration.pptx?raw=true)

Reply:
done, will be in PR